src/Add.js

```diff
@@ -0,0 +1,69 @@
+export default class AddList {
+    constructor() {
+      const previousData = localStorage.getItem('todoData');
+      this.todoDetails = previousData ? JSON.parse(previousData) : [];
+    }
+  
+    displayList() {
+      const reciveData = localStorage.getItem('todoData');
+      if (reciveData && JSON.parse(reciveData).length > 0) {
+        this.todoDetails = JSON.parse(reciveData);
+        const row = document.getElementById('lists');
+        row.innerHTML = '';
+        for (let i = 0; i < this.todoDetails.length; i += 1) {
+          row.innerHTML += `<li class="removeLine">
+                                <div class="rowData"><input class="edit-text" type="checkbox" ${
+    this.todoDetails[i].completed ? 'checked' : ''
+  } />
+                                <input class="editBtn" type="text" value="${this.todoDetails[i].title}" data-index="${i}" readonly /></div>
+                                <button id="${this.todoDetails[i].index}" class="remove-btn"><i class="fas fa-trash"></i></button>
+                              </li><hr>`;
+        }
+        const removeBtn = document.querySelectorAll('.remove-btn');
+        const removeLine = document.querySelectorAll('.removeLine');
+  
+        for (let i = 0; i < removeBtn.length; i += 1) {
+          removeBtn[i].addEventListener('click', () => {
+            removeLine[i].remove();
+            this.deleteRow(i);
+            this.displayList();
+          });
+        }
+  
+        const editBtn = document.querySelectorAll('.editBtn');
+        editBtn.forEach((editElement) => {
+          editElement.addEventListener('click', () => {
+            const index = editElement.getAttribute('data-index');
+            const editInput = document.querySelector(`input.editBtn[data-index="${index}"]`);
+            editInput.readOnly = false;
+            editInput.addEventListener('blur', (event) => {
+              const newTitle = event.target.value;
+              this.updateRowTitle(index, newTitle);
+              event.target.readOnly = true;
+            });
+          });
+        });
+      }
+    }
+  
+    addRow(title, completed, index) {
+      const todo = { title, completed, index };
+      this.todoDetails.push(todo);
+      localStorage.setItem('todoData', JSON.stringify(this.todoDetails));
+    }
```

Comment:
According to the best practices stated in [this article for clean code](https://devinduct.com/blogpost/22/javascript-clean-code-best-practices), it is best to have not more than 2 arguments in a function as this makes code testing a lot easier. I kindly suggest that you look into following this guideline.